src/home/index.html

```diff
@@ -8,268 +8,308 @@
+        <div class="card-4">
+          <div class="card-1-1">
+            <h2>Tonic</h2>
+            <ul>
+              <li class="cano">CANOPY</li>
+              <li class="dot empty"></li>
+              <li class="dot">Back End Dev</li>
+              <li class="dot empty"></li>
+              <li class="dot">2015</li>
+            </ul>
+            <p class="desc">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="skills">
+              <li>html</li>
+              <li>css</li>
+              <li>javascript</li>
+            </ul>
+            <button type="button" class="btn-tonic">See project</button>
+          </div>
+          <div class="card-1-2">
+            <div class="image-container4"></div>
+          </div>
         </div>
-        <div class="language-title">
-          <p>Language</p>
-          <img src="../assert/images/chevron-h.png" alt="language" />
+      </section>
+      <section class="about">
+        <div class="text-home-about">
+          <h1 class="heading heading-about">About Mwafrika</h1>
+          <h3 class="description-about">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
+          </h3>
+          <h3 class="subdescription">LET'S CONNECT</h3>
+
+          <div class="social-container social-container-about">
+            <ul>
+              <li class="social">
+                <a href="#">
+                  <img src="../assert/images/twitter.png" alt="Twitter" />
+                </a>
+              </li>
+              <li class="social">
+                <a href="#">
+                  <img src="../assert/images/Linkedin.png" alt="LinkedIn" />
+                </a>
+              </li>
+              <li class="social">
+                <a href="#">
+                  <img src="../assert/images/m.png" alt="Medium" />
+                </a>
+              </li>
+              <li class="social">
+                <a href="#">
+                  <img src="../assert/images/github.png" alt="Github" />
+                </a>
+              </li>
+              <li class="social">
+                <a href="#">
+                  <img src="../assert/images/fingers.png" alt="Finger" />
+                </a>
+              </li>
+            </ul>
+            <button type="button" class="btn-tonic">See project</button>
+          </div>
         </div>
-        <div class="langage-list">
+
+        <div class="about-img">
+          <!-- <img src="../assert/images/try.svg" alt="About" /> -->
+          <div class="lang title">
+            <h2>Languages</h2>
+            <img src="../assert/images/chevron-v.png" alt="Language" />
+          </div>

```

Comment:
- [ ] Hello please notice and try to use the correct `font-color` and `font-weight` for the text "Languages"

|Figma Design|Your Design|
|---|---|
|![image](https://user-images.githubusercontent.com/18730106/149633875-74e31de3-d0e0-4ee3-abec-54519b0de53f.png)|![image](https://user-images.githubusercontent.com/18730106/149633893-1d9a19f3-1ba4-4dea-9e40-bf4b70416644.png)|

- [ ] Please make the same change for the other text in this section (that is the "Frameworks" and "Skills" text).

|Figma Design|Your Design|
|---|---|
|![image](https://user-images.githubusercontent.com/18730106/149633947-0b91e59d-0dcb-49cf-830e-4424681efb52.png)|![image](https://user-images.githubusercontent.com/18730106/149633960-16355058-10bb-44e2-8f3b-b3b24289b1a6.png)|

Reply:
Yeah. I've done already, you can have a look sir